rust_snuba/Cargo.toml

```diff
@@ -45,6 +45,11 @@ tracing = "0.1.40"
 tracing-subscriber = { version = "0.3.18", features = ["env-filter", "json"] }
 uuid = "1.5.0"
 parking_lot = "0.12.1"
+sentry_usage_accountant = "0.0.3"
+
+
+[patch.crates-io]
+rdkafka = { git = "https://github.com/getsentry/rust-rdkafka", branch = "base-consumer", features = ["cmake-build", "tracing"] }
```

Comment:
Is this solving the dependency issue we saw yeaterday? (dependency on rdkafka 0.29.0)

Reply:
yes, i recommended this. it makes the dependency resolver pretend that "rdkafka from crates.io" actually comes from this repo. this is useful because cogs recorder depends on "rdkafka from crates.io", not "rdkafka from random git repo", so without this change cargo will attempt to build two separate versions of rdkafka, one for the cogs recorder, and one for the application. normally this works out great, particularly for things like `serde` or `log`, but for a library linking to a C++ codebase it plainly breaks at buildtime.